File-Formats/DocIO/word-to-image.md

```diff
@@ -332,6 +332,319 @@ End Using
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-Image-conversion/Custom-image-resolution).
 
+## Fallback fonts
+
+During Word to Image conversions, if a glyph of the input text is unavailable in the specified font, the text will not be rendered properly. To address this, the Syncfusion Word (DocIO) library allows users to specify fallback fonts. When a glyph is missing, the library will use one of the fallback fonts to render the text correctly in the output image document.
+
+Users can configure fallback fonts in the following ways:
+* Initialize default fallback fonts.
+* Set custom fonts as fallback fonts for specific script types, including Arabic, Hebrew, Chinese, and Japanese etc.
+* Set custom fonts as fallback fonts for a particular range of Unicode text.
+
+N> DocIO internally uses fallback fonts for Unicode characters. Therefore, the specified fallback fonts must be installed in the production environment or embedded in the input Word document (DOCX). Otherwise, it will not render the text properly using the fallback fonts.
```

Comment:
@LokeshBaskar - Is this statement correct? 
`DocIO internally uses fallback fonts for Unicode characters.`

Reply:
Yes, Modified sentences to understand better.